R/run_model.R

```diff
@@ -108,6 +110,27 @@ build_output_metadata <- function(output) {
   )
 }
 
+build_output_table_metadata <- function() {
+  list(
+    presets = list(
+      list(
+        label = scalar(t_("TABLE_SEX_BY_AREA")),
+        column = scalar("sex"),
+        row = scalar("area_id")
```

Comment:
actually have been using this and testing the waters and I just realised, this needs to be `area`. `area_id` is the `column_id` of the filter not the `id`!
```suggestion
        row = scalar("area")
```